zerver/openapi/zulip.yaml

```diff
@@ -17600,6 +17600,14 @@ paths:
                           This Zulip server's configured minimum required length for passwords.
                           Necessary for password change UI to show whether the password
                           will be accepted.
+                      password_max_length:
+                        type: integer
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          This Zulip server's configured maximum length for passwords.
+                          Necessary for password change UI to show whether the password
+                          will be accepted.
```

Comment:
Every API change should always document when it happened in two places, the changelog and the actual documentation.

``` diff
diff --git a/zerver/openapi/zulip.yaml b/zerver/openapi/zulip.yaml
index 5abd7e101c..ebaecfe210 100644
--- a/zerver/openapi/zulip.yaml
+++ b/zerver/openapi/zulip.yaml
@@ -17608,6 +17608,8 @@ paths:
                           This Zulip server's configured maximum length for passwords.
                           Necessary for password change UI to show whether the password
                           will be accepted.
+
+                          **Changes**: New in Zulip 10.0 (feature level 338).
                       password_min_guesses:
                         type: integer
                         description: |
```